app/controllers/application_controller.rb

```diff
@@ -3,6 +3,10 @@ class ApplicationController < ActionController::Base
   # before_action :authenticate_user!
   before_action :configure_permitted_parameters, if: :devise_controller?
 
+  def current_trip
+    @trip || Trip.new(session[:trip])
```

Comment:
To be removed for now and put back in once sessions are relevant for further progress